package/libyui.spec

```diff
@@ -148,29 +113,27 @@ install -m0755 -d $RPM_BUILD_ROOT/%{_docdir}/%{bin_name}/
 install -m0755 -d $RPM_BUILD_ROOT/%{_libdir}/yui
 install -m0644 ../COPYING* $RPM_BUILD_ROOT/%{_docdir}/%{bin_name}/
 
-%clean
-rm -rf "$RPM_BUILD_ROOT"
 
 %post -n %{bin_name} -p /sbin/ldconfig
-
 %postun -n %{bin_name} -p /sbin/ldconfig
 
+
 %files -n %{bin_name}
 %defattr(-,root,root)
-%dir %{_libdir}/yui
-%dir %{_datadir}/libyui
 %{_libdir}/lib*.so.*
 %doc %dir %{_docdir}/%{bin_name}
 %license %{_docdir}/%{bin_name}/COPYING*
 
+
 %files devel
 %defattr(-,root,root)
 %dir %{_docdir}/%{bin_name}
 %{_libdir}/lib*.so
-%{_prefix}/include/yui
-%{_libdir}/pkgconfig/%{name}.pc
-%{_libdir}/cmake/%{name}
+%{_includedir}/yui
+%dir %{_datadir}/libyui
 %{_datadir}/libyui/buildtools
 %doc %{_docdir}/%{bin_name}/examples
+%{_libdir}/pkgconfig/%{name}.pc
+# %{_libdir}/cmake/%{name}
```

Comment:
leftover?

Reply:
No, that might come back once we merge all the libyui repos. Right now we have quite some duplication in the CMakeLists.txt files, so in the long run we might choose to move some of that out to .cmake files that we can then include.

For this iteration, I opted for clarity at the cost of some amount of duplication.